Gemfile.lock

```diff
@@ -303,9 +303,6 @@ DEPENDENCIES
   jbuilder (~> 2)
   m (~> 1)
   minitest (= 5.6.0)
-  net-imap
```

Comment:
Why did this change? 🤔

Reply:
Seems related to https://github.com/rails/rails/pull/44083